src/api/schemas.py

```diff
@@ -1,8 +1,9 @@
 import urllib
 from datetime import date
-from typing import Dict, Optional
+from typing import Dict, List, Optional
 
-from pydantic import BaseModel, Extra, Field, NonNegativeInt, StrictStr, field_validator, root_validator
+from pydantic import (BaseModel, Extra, Field, NonNegativeInt, StrictStr,
+    field_validator, root_validator, conlist)
```

Comment:
У нас такой коде стайл: 
```suggestion
from pydantic import (
    BaseModel,
    Extra,
    Field,
    NonNegativeInt,
    StrictStr,
    field_validator,
    root_validator,
    conlist,
)
```

Reply:
не заметил этого стиля)) 
сделаю